Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -441,9 +441,9 @@ final class IssueCommentSectionController:
             ToastManager.showGenericError()
             return
         }
-
+        
         let alertTitle = NSLocalizedString("%@'s comment", comment: "Used in an action sheet title, eg. \"Basthomas's comment\".")
-
+        
```

Comment:
Can we remove this? Make sure to have your Xcode settings:

<img width="797" alt="screen shot 2018-05-08 at 10 42 10 pm" src="https://user-images.githubusercontent.com/739696/39792932-203fb682-5311-11e8-9a0e-34d007e3063c.png">